notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -336,13 +336,14 @@
       "source": [
```

Comment:
<p>Add text cell explaining</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>I did that in your last review of this code last week. :)</p>